### PR TITLE
Watchlater

### PR DESCRIPTION
**Pull request by sharmayatendra**
# Changes

- watch later functionality

image

### CODE REVIEW

File: src/context/watchlater-context.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    )
+}
+
+const addToWatchLater = async (video,token,setWatchlaterVideos,navigate) => {
+    if(token) {
+        try{
+            const resp = await axios.post('/api/user/watchlater' , {video} , {
+              headers: {authorization: token}
+            })
+            console.log(resp);
```
Comment (tushargoyal812) [inline src/context/watchlater-context.js:21]: remove console

File: src/context/watchlater-context.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    
+  }
+
+const removeFromWatchLater = async (token,setWatchlaterVideos,videoId,navigate) => {
+    if(token) {
+        try{
+            const resp = await axios.delete(`/api/user/watchlater/${videoId}` , {
+                headers: {authorization: token}
+            })
+            console.log(resp);
```
Comment (tushargoyal812) [inline src/context/watchlater-context.js:38]: remove console

File: src/pages/VideoDetails/VideoDetails.jsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    const { likeVideo, setLikeVideo} = useLike()
+    const {watchlaterVideos,setWatchlaterVideos} = useWatchlater()
+    const currentVideoPlaying = videos.find(item => item._id === videoId)
+
+    const addToLiked = async (video) => {
+      try {
+        const resp = await axios.post('/api/user/likes' , {video}, {
+          headers: {authorization: token}
+        })
+        console.log(resp);
```
Comment (tushargoyal812) [inline src/pages/VideoDetails/VideoDetails.jsx:26]: remove console

**Review by yashpatel1797 — APPROVED** [review]
code looks clean
file structure is good
design looks consistent

**Pull request merged**